microcosm_postgres/tests/temporary/test_temporary.py

```diff
@@ -83,7 +83,7 @@ def test_upsert_into_on_conflict_do_update(self):
                             index_elements=["id"],
                             set_=dict(
                                 type=CompanyType.public,
-                                updated_at=datetime.now(),
+                                updated_at=datetime.utcnow(),
```

Comment:
we always use UTC in db stored datetimes, this test was wrongly using local time. this worked on CI since it is configured to use UTC but would fail when running locally